### DISCUSSION TITLE
How long does it typically take to update?

### DISCUSSION THREAD

**Discussion by Parma-John** [philosowaffle/peloton-to-garmin#418] [General]
I tried to update the app to version 3.3.1 and it seems to run just fine but when I go to my Garmin connect the workouts aren't showing up its been like 15-20 mins. I'll check again in a bit but maybe I messed something up.

**Comment by philosowaffle** [comment]
1. Are you running P2G via the WIndows Exe, Docker, or Github actions?
2. Could you also share your log output?

**Reply by Parma-John** [reply]
<img width="861" alt="image" src="https://github.com/philosowaffle/peloton-to-garmin/assets/106790854/cb97164f-c5db-4a23-a9e5-22d7401802f8">
 
I don't know if you waned a screenshot but I can't seem to get it to link up and I have checked the login info of both accounts like 10 times. I even tried deleting the whole folder redownloading it and resetting up the config file but I got the same results.
